README.md

```diff
@@ -53,41 +53,101 @@ There are plenty of supported formats and we constantly work on adding new.
 
 ## Using 
 
+Feel free to read ["How to scan barcodes in Qt Qml application" blog post](https://scythe-studio.com/blog/how-to-scan-barcodes-in-qt-qml-application) if you are interested in the details behind scanning mechanism. Also you can try out the QmlReaderExample in order to see how it is working.
+
+SCodes supports generating barcodes as well. It is covered in ["How to generate barcode in Qt Qml" application blog post](https://scythe-studio.com/en/blog/how-to-generate-barcode-in-qt-qml-application).  Also you can try out the QmlGeneratorExample in order to see how it is working.
+
+Above blog posts contains step by step tutorial on how to do that for Qt5 version. For Qt6 ([see here](#porting)).
+
 ### QMake
 All you need to do is to follow these steps.
 
 1. Add SCodes as submodule, by typing `git submodule add git@gitlab.com:scythestudio/scodes.git`
 2. Update submodule `git submodule update --recursive --init` (you can also put wrapper files to your project manually without adding submodule)
-3. Add `include(scodes/src/SCodes.pri` to your .pro file
-4. Import SCodes in your Qml file `import com.scythestudio.scodes 1.0`
-5. Import multimedia module `import QtMultimedia 5.15`
-6. If build fails, try to add `CONFIG += c++17` to your .pro file
-7. You are done. Get inspired by [QML Barcode Reader demo](https://github.com/scytheStudio/SCodes/blob/master/examples/QmlBarcodeReader/qml/ScannerPage.qml) to test wrapper.
+3. Add `include(scodes/src/SCodes.pri)` to your .pro file
+4. If you want to use barcode reader functionality you need to register `SBarcodeFilter` class for Qt5 and `SBarcodeScanner` class for Qt6. For both version, seperate them with if directive to register([how to register reader class](#register-reader)). As for barcode generator functionality you just need to register `SBarcodeGenerator` class([how to register generator class](#register-generator)).
```

Comment:
Wouldn't it be better to register the class under the same `SBarcodeScanner` name?
Now it's confusing a bit.

Reply:
That classes already registered as SBarcodeFilter but I missed to update the readme file. SBarcodeScanner is  more proper name for both ofcourse, fixed!